product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -100,17 +100,17 @@ __OUTPUT__
 ```
 
 This command runs the `vacuum full foo` command on the node-one and node-three nodes.
-Note that the node names are passed to the function in an array.
+The node names are passed to the function in an array.
 
-The bdr.run_on_nodes reports its results as JSONB. 
-The results include the name of the node and the response (or error message)  resulting from running the command.
-Other fields may be included may not be relevant. 
+The `bdr.run_on_nodes` function reports its results as JSONB. 
+The results include the name of the node and the response (or error message) resulting from running the command.
+Other fields included might not be relevant. 
 
-The results also appear as a single string which is hard to read. To make it more readable, we can apply some formatting.
+The results also appear as a single string that's hard to read. To make it more readable, apply some formatting.
```

Comment:
We are about to go through the steps to apply formatting. You've changed it such that it suggests the reader should go and work out how to apply formatting.

Reply:
Okay, I'm not sure what you mean. Can you revise so it doesn't use first person?

Reply:
By applying some formatting to this string, it can become more readable.

Reply:
```suggestion
The results also appear as a single string that's hard to read. By applying some formatting to this string, it can become more readable.
```